### ISSUE TITLE
Feature Request: Add priority option

### ISSUE THREAD

**Issue by ghost** [farfromrefug/sorttv#2]
My movies keep getting sorted in my TV Shows. I feel that if the movies were sorted first, this wouldn't happen. A configurable option would be great.

**Comment by farfromrefug** [comment]
wow still using this? :D I have not for years now. You should look at sonarr ;)

**Comment by ghost** [comment]
Yeah, and I plan on using it for many more years because it's simple and I love it.

**Comment by farfromrefug** [comment]
well i am quite glad to hear that. Do you think you can make the modification yourself ? i can't really test it anymore
